### ISSUE TITLE
Fix Program Areas filter for `Diversity, Equity & Inclusion`

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4120]
### Dependency
- #4118

### Overview
We need to fix the Program Areas filter on the [projects page](https://www.hackforla.org/projects/) so that website visitors can filter and find the projects with the `Diversity, Equity & Inclusion` program area.

<details>

<summary>Demonstration of the bug</summary>

![filter bug](https://user-images.githubusercontent.com/88953806/225777404-06bcad4d-4d07-4141-b21b-f9b2191e8f87.gif)

</details>

### Action Items
- [x] In `assets/js/current-projects.js`, fix the Program Areas filter on the [projects page](https://www.hackforla.org/projects/) so that it works for `Diversity, Equity & Inclusion`.

#### Merge Team, Technical Lead, or Product Manager
- [ ] Once the pull request associated with this issue is merged, go to issue #3328 and check if all the issues listed under the `Dependency` section are closed. If they are all closed, move that issue (#3328) into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.

### Resources/Instructions
- [projects page](https://www.hackforla.org/projects/)
- [assets/js/current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js)
- [pages/projects-page.html](https://github.com/hackforla/website/blob/gh-pages/pages/projects-page.html)
- [_includes/current-projects.html](https://github.com/hackforla/website/blob/gh-pages/_includes/current-projects.html)

**Comment by JessicaLucindaCheng** [comment]
Once this issue is in the New Issue Approval column, a dev lead needs to review the issue and determine if it is ready for a PM to add a milestone and prioritize it. If it isn't ready, a dev lead will need to edit this issue so that it is ready for a PM to add a milestone and prioritize it.

**Comment by jdingeman** [comment]
@JessicaLucindaCheng - just added a gif, sending this for Milestone.

**Comment by JessicaLucindaCheng** [comment]
@jdingeman You said you added a gif. I don't see it. Sending this back to dev leads.

**Comment by ExperimentsInHonesty** [comment]
@jdingeman @JessicaLucindaCheng once you all sort out the missing .gif, then go ahead and prioritize this (please note I added the time sensitive label)

**Comment by jdingeman** [comment]
@ExperimentsInHonesty @JessicaLucindaCheng - oh that's embarrassing. I forgot to update the comment the first time.

**Comment by t-will-gillis** [comment]
Availability: Tues 3/21
ETA: Tues 3/21 5 pm

**Comment by ExperimentsInHonesty** [comment]
@t-will-gillis 

> @ExperimentsInHonesty @JessicaLucindaCheng
> I am working on https://github.com/hackforla/website/issues/4120 - Yes "Diversity / Equity / Inclusion" will work - I just checked.
> 
> (The previous "Diversity, Equity & Inclusion" had problems not only because of the "&" but also because of the ",". Even removing the ampersand, with the comma the dropdown was splitting into two unique program areas of "Diversity" and "Equity and Inclusion" . )

Perfect!  Let's make that change (change the Program Area to be Diversity / Equity / Inclusion) on this issue.  Which solves the problem, It's a better solution than editing the script.

**Comment by t-will-gillis** [comment]
@ExperimentsInHonesty @JessicaLucindaCheng While I am changing the cards to Diversity / Equity / Inclusion, do we want to change the 'Join Us' page (it needs a comma at least...):
![join us ](https://user-images.githubusercontent.com/40799239/227040441-142a60e6-5d7b-490e-b46c-fc786def0017.png)
do we want to change the 'About' page: 
![about](https://user-images.githubusercontent.com/40799239/227040815-1c8d3c10-6a60-4fb1-add5-5dd265ec958f.png)
and finally the title on the Program Areas page:
![after_program_areas](https://user-images.githubusercontent.com/40799239/227042113-bd13b757-5980-4d1c-b496-f87c113cb008.png)

**Comment by ExperimentsInHonesty** [comment]
@t-will-gillis Go ahead and [make an ER](https://github.com/hackforla/website/issues/new?assignees=&labels=size%3A+0.25pt&template=emergent-request.md&title=ER%3A+%5Breplace+with+info+%5D+) for updating it to say Diversity, Equity & Inclusion every place that does not use a dropdown or other  feature that would make that non feasible.

**Comment by ExperimentsInHonesty** [comment]
@t-will-gillis are you putting in a pr on this issue to change the text in the project.md files from 
```
program-area: 
  - Diversity, Equity & Inclusion
```
to 
```
program-area: 
  - Diversity / Equity / Inclusion
  ```

for the following projects?
- New Schools Today
- Write for All

Or are you making new issues for those two files and the changes?

**Comment by t-will-gillis** [comment]
@ExperimentsInHonesty PR #4270 is in review currently and will change both `new-schools-today.md` and `writeforall.md` to:
```
program-area: 
  - Diveristy / Equity / Inclusion
``` 
Then there are three separate issues for changing program area to be `Diversity, Equity & Inclusion` everywhere else if  it will not cause a bug:

- #4284 
- #4286 
- #4287

Is this what you are asking?
